next.config.js

```diff
@@ -57,9 +57,14 @@ const nextConfig = {
         destination: '/data-availability',
         permanent: true,
       },
+      {
+        source: '/files/:slug',
+        destination: '/papers/:slug',
+        permanent: true,
+      },
```

Comment:
I think we can drop this redirect and use cdn links on the components repo

Reply:
I'll add a to-do to remove this in a future PR. Keeping these supports telling a browser to transition to using the permanent new URL. In case there are saved links to `/files/*`